spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkTable.java

```diff
@@ -131,12 +133,12 @@ public SparkTable(Table icebergTable, boolean refreshEagerly) {
   public SparkTable(Table icebergTable, String branch, boolean refreshEagerly) {
     this(icebergTable, refreshEagerly);
     this.branch = branch;
+    final Snapshot snapshot = icebergTable.snapshot(branch);
```

Comment:
Minor: The community doesn't use `final` for local vars.

Reply:
I updated it. Thanks.